src/index.js

```diff
@@ -25,19 +27,35 @@ async function runTest(files, reporter, requireFilePath) {
             allTestFiles.push(requireFilePath);
         }
         allTestFiles.push(...files);
-        await brs.execute(allTestFiles, {
+        let returnVals = await brs.execute(allTestFiles, {
             stdout: reporterStream,
             stderr: process.stderr
         });
+
         reporterStream.end();
+
+        if (forbidFocused && returnVals.length > 0) {
+            // iterate through return values and see if there are focused cases
+            for (output of returnVals) {
+                let focusedFiles = output.getValue().get("fileswithfocusedcases").getElements();
+                if (focusedFiles.length > 0) {
+                    let formattedList = focusedFiles.map((brsStringPath) => `    ${brsStringPath.value}`).join("/n");
```

Comment:
does the formatted list work if you use a `\t` instead of the white spaced? also, shouldn't  the last part be `join(\/n");`

Reply:
+1 to using `\t`